main.js

```diff
@@ -9,15 +11,44 @@ let game = {
     cardsTurn: [],
     // Stores the number of clicks in the current turn 0 - 2
     clicks: 0,
-    turn: 0
+    turn: 0,
+    timer: 0,
+    timerSwitch: 0,
```

Comment:
It looks like this is being used as a true/false, why not use a boolean?

Reply:
changed to a bool